FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/BaseHealthChecker.cs

```diff
@@ -44,15 +48,19 @@ public async Task<HealthCheckResult> PerformHealthCheckAsync(CancellationToken c
                     Status = HealthCheckStatus.UNHEALTHY,
                     ErrorMessage = $"Unhandled exception : {e.Message}.",
                 };
-                _logger.LogError($"Unhandled exception occured in health check component {Name}. {e.Message}");
+
+                _diagnosticLogger.LogWarning($"Unhandled exception occured in health check component {Name}. {e.Message}");
```

Comment:
When we use `logWarning`?

Reply:
Only health check will log warning if some component is unhealthy.